src/main/java/teammates/ui/webapi/request/CourseBasicRequest.java

```diff
@@ -0,0 +1,23 @@
+package teammates.ui.webapi.request;
+
+import teammates.ui.webapi.output.CourseData;
+
+/**
+ * The basic request of modifying a course.
+ */
+public class CourseBasicRequest extends BasicRequest {
+
+    private CourseData storedData;
```

Comment:
> * Added CourseBasicRequest and their corresponding requests. I reused CourseInfo in CourseBasicRequest instead of re-declaring those variables like what FeedbackSessionBasicRequest because this way if CourseInfo changes the others change as well.

I don't think this is a good reason to "reuse" course data.
- The wrapper gives extra confusion to other developers and does not follow the convention of other APIs.
- It is just in this case that the request and response have the same fields.